examples/persistent-volumes/volumes/local-01.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolume
+apiVersion: v1beta3
+metadata:
+  name: pv0001
+  labels:
+    type: local
+spec:
+  capacity:
+    storage: 10Gi
```

Comment:
The thing that you're going to need to be careful of here is that there are highly likely to be multiple storage resource dimensions here over time, primarily:
1. space, 
2. IOPS
3. bandwidth

Typically you will want to carefully control oversubscription of all of those.